compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -230,6 +230,9 @@ error.invalid.worker.join.result.type=\
 error.invalid.worker.timeout.result.type=\
    invalid worker timeout result type, expected 'map'
 
+error.invalid.worker.reference=\
+   invalid worker reference ''{0}''
```

Comment:
It is not invalid, but we don't support it yet. Maybe we can reword this, saying. "worker reference not supported here yet."

Reply:
unsupported worker reference ''{0}'' ?

Reply:
+1